activesupport/test/core_ext/numeric_ext_test.rb

```diff
@@ -389,4 +389,16 @@ def test_to_s__injected_on_proper_types
   def test_in_milliseconds
     assert_equal 10_000, 10.seconds.in_milliseconds
   end
+
+  def test_positive
```

Comment:
I'm waiting a response from @rafaelfranca then I'll change the behaviour and add tests for all numeric types.